src/config.json

```diff
@@ -1,4 +1,5 @@
 {
   "STATIC_DATABASE_URL": "http://localhost:3001",
-  "DATABASE_URL": "http://localhost:9000"
+  "DATABASE_URL": "http://localhost:9000",
+  "LOCAL_DATABASE_URL": "http://localhost:8080"
 }
```

Comment:
With the latest version of backend, I've had to move the port to 9000, as jenkins runs on 8080